x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/openai/embeddings/OpenAiEmbeddingsTaskSettingsTests.java

```diff
@@ -58,7 +62,7 @@ public void testFromMap_MissingModel_ThrowException() {
     public void testFromMap_CreatesWithModelAndUser() {
         var taskSettings = OpenAiEmbeddingsTaskSettings.fromMap(
             new HashMap<>(Map.of(OpenAiEmbeddingsTaskSettings.OLD_MODEL_ID_FIELD, "model", OpenAiEmbeddingsTaskSettings.USER, "user")),
-            false
+            OpenAiParseContext.PERSISTENT
```

Comment:
is there a reason all of these tests use PERSISTENT?

Reply:
Yeah not really, I'll move the logic that relies on the context into a static function and test that.